src/View/Helper/DatatableHelper.php

```diff
@@ -48,52 +51,83 @@ class DatatableHelper extends Helper
     ];
 
     private $columnSearchTemplate = <<<COLUMN_SEARCH_CONFIGURATION
-        
+
         var api = this.api();
 
+        var columnsSearch = %s;   
+
         // For each column
         api
-            .columns()
-            .eq(0)
-            .each(function (colIdx) {
-                // Set the header cell to contain the input element
-                var cell = $('.filters th').eq(
-                    $(api.column(colIdx).header()).index()
-                );
-                var title = $(cell).text();
-                $(cell).html('<input type="text" style="width: 100%%;" placeholder="' + title + '" />');
-
-                // On every keypress in this input
-                $(
-                    'input',
-                    $('.filters th').eq($(api.column(colIdx).header()).index())
-                )
-                .off('keyup change')
-                .on('keyup change', function (e) {
-                    e.stopPropagation();
-
-                    // Get the search value
-                    $(this).attr('title', $(this).val());
-                    var regexr = '({search})'; //$(this).parents('th').find('select').val();
-
-                    var cursorPosition = this.selectionStart;
-                    // Search the column for that value
-                    api
-                        .column(colIdx)
-                        .search(
-                            this.value != ''
-                                ? regexr.replace('{search}', '(((' + this.value + ')))')
-                                : '',
-                            this.value != '',
-                            this.value == ''
+        .columns()
```

Comment:
tab change here hiding real changes done in the MR, please use the same code style